spec/android_spec.rb

```diff
@@ -86,7 +86,7 @@
      YScale: 0.500\r\n        XPrecision: 2.000\r\n        YPrecision: 2.000\r\n
 _______________________________________________________
         device = DeviceAPI::Android::Device.new(serial: 'SH34RW905290')
-        allow(Open3).to receive(:capture3) { [out, '', $STATUS_ZERO] }
+        allow(Open3).to receive(:capture3) { [out, '', STATUS_OK] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.